usersimcrs/items/ratings.py

```diff
@@ -133,13 +166,16 @@ def get_random_user_id(self) -> str:
         return random.choice(list(self._user_ratings.keys()))
 
     def create_split(
-        self, historical_ratio: float
+        self,
+        historical_ratio: float,
+        sampler: Callable = user_item_sampler,
     ) -> Tuple["Ratings", "Ratings"]:
         """Splits ratings into historical and ground truth ratings.
 
         Args:
             historical_ratio: Ratio ([0..1]) of ratings to be used as historical
               data.
+            sampler: Callable performing the sampling.
```

Comment:
Nit: "... the sampling of items per user."

Reply:
Done.